docs/source/languages/orquesta.rst

```diff
@@ -107,16 +107,103 @@ execution, the task is evaluated for completion. If criteria for transition is m
 set of tasks is invoked, sequentially in the order of the transitions and tasks that are listed.
 
 If more than one tasks transition to the same task and ``join`` is specified in the latter (i.e. the
-task named "task3" in the example below), then the task being transitioned into becomes a barrier
-for the inbound task transitions. There will be only one instance of the barrier task. In the
-workflow graph, there will be multiple inbound edges to the barrier node.
+task named ``barrier_task`` in the example below), then the task being transitioned into becomes a
```

Comment:
Please remove `(i.e. the task named “task3” in the example below)` as this refers to a different example.  Since you include an example here, let's remove the wrong reference here.